README.md

```diff
@@ -470,12 +470,12 @@ Routermaster provides monitoring endpoints:
 - `<name>`: the names of all topics routed into this subscriptions queue.
 - `<sent_count>`: total number of events ever sent on this topic.
 - `<queue_size>`: current number of events in the subscription queue.
-- `<oldest>`: timestamp (seconds since epoch) of the oldest pending event.
+- `<staleness>`: timestamp (seconds since epoch) of the oldest pending event.
```

Comment:
this one is to be consistent with the ones above. the other ones are explaining the values of the params while this one was explaining the key of the param.